2-mandatory/2-oxygen-levels.js

```diff
@@ -11,7 +11,13 @@
     Some string methods that might help you here are .replace() and .substring().
 */
 
-function findSafeOxygenLevel() {}
+function findSafeOxygenLevel(oxygenLevel) {
```

Comment:
I'm not sure this is really working. 

Take a look at the tests to see what sort of inputs you will need to handle. I think that the inputs are strings, so to check they are greater or less than a number, you will need to do some conversion first.

Reply:
working now, thank you